Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Raw_image_dialog.cpp

```diff
@@ -26,3 +34,18 @@ unsigned int Raw_image_dialog::image_word_size() const {
     return 1;
 }
```

Comment:
Here I can understand why the type was `int`: the possible values are only 1 or 2. But the change is right too, to avoid a conversion warning I suppose?